cli/command/secret/create.go

```diff
@@ -45,7 +45,7 @@ func runSecretCreate(dockerCli *command.DockerCli, options createOptions) error
 
 	secretData, err := ioutil.ReadAll(os.Stdin)
 	if err != nil {
-		return fmt.Errorf("Error reading content from STDIN: %v", err)
+		return fmt.Errorf("error reading content from STDIN: %v", err)
```

Comment:
Looking at this again; is something printed before `error` on the CLI? If not, I think it's okay to have it start with a capital `E`

@vdemeester ?